pkg/minishift/addon/parser/addon_parser_test.go

```diff
@@ -109,216 +109,138 @@ func Test_successful_parsing_of_addon_dir_without_remove_addon_file(t *testing.T
-	}
+	assert.Equal(t, expectedName, addOn.MetaData().Name())
 
 	expectedNumberOfCommands := 2
-	if len(addOn.Commands()) != expectedNumberOfCommands {
-		t.Errorf("Unexpected number of commands. Found %d, but expected %d", len(addOn.Commands()), expectedNumberOfCommands)
-	}
+	assert.Len(t, addOn.Commands(), expectedNumberOfCommands)
 
 	expectedNumberOfRemoveCommands := 2
-	if len(addOn.RemoveCommands()) != expectedNumberOfRemoveCommands {
-		t.Errorf("Unexpected number of commands. Found %d, but expected %d", len(addOn.RemoveCommands()), expectedNumberOfRemoveCommands)
-	}
+	assert.Len(t, addOn.RemoveCommands(), expectedNumberOfRemoveCommands)
 
 	_, ok := addOn.Commands()[0].(*command.OcCommand)
-	if !ok {
-		t.Fatalf("Expected an OcCommand. Got %s", reflect.TypeOf(addOn.Commands()[0]).Name())
-	}
+	assert.True(t, ok)
 }
 
 func Test_empty_lines_and_comments_in_content_are_ignored(t *testing.T) {
 	_, commands, err := testParser.parseAddOnContent(strings.NewReader(addOnWithEmptyLinesAndCommments))
-
-	if err != nil {
-		t.Fatal("Unexpected error parsing addon content: " + err.Error())
-	}
+	assert.NoError(t, err)
 
 	expectedNumberOfCommands := 3
-	if len(commands) != expectedNumberOfCommands {
-		t.Errorf("Unexpected number of commands. Found %d, but expected %d", len(commands), expectedNumberOfCommands)
-	}
+	assert.Len(t, commands, expectedNumberOfCommands)
 }
 
 func Test_multiple_lines_in_description_with_tab_and_space(t *testing.T) {
 	meta, commands, err := testParser.parseAddOnContent(strings.NewReader(addOnWithMultilineDescriptionWithTabAndSpace))
-
-	if err != nil {
-		t.Fatal("Unexpected error parsing addon content: " + err.Error())
-	}
+	assert.NoError(t, err)
 
 	expectedDescription := []string{"This is one line", "This is second line", "This is third line", "This is forth line"}
 	minishiftTesting.AssertEqualSlice(expectedDescription, meta.Description(), t)
 
 	expectedNumberOfCommands := 1
-	if len(commands) != expectedNumberOfCommands {
-		t.Errorf("Unexpected number of commands. Found %d, but expected %d", len(commands), expectedNumberOfCommands)
-	}
+	assert.Len(t, commands, expectedNumberOfCommands)
 }
 
 func Test_multiple_lines_in_description_with_comment(t *testing.T) {
 	meta, commands, err := testParser.parseAddOnContent(strings.NewReader(addOnWithMultilineDescriptionWithComment))
-
-	if err != nil {
-		t.Fatal("Unexpected error parsing addon content: " + err.Error())
-	}
+	assert.NoError(t, err)
 
 	expectedName := "foo"
-	if meta.Name() != expectedName {
-		t.Errorf("Expected Name: %s, Got: %s", expectedName, meta.Name())
-	}
+	assert.Equal(t, expectedName, meta.Name())
 
 	expectedDescription := []string{"This is one line", "This is:second line", "This is third line"}
-	minishiftTesting.AssertEqualSlice(expectedDescription, meta.Description(), t)
-
+	//minishiftTesting.AssertEqualSlice(expectedDescription, meta.Description(), t)
```

Comment:
why is this commented out?

Reply:
Sorry, this was supposed to be removed, got left there, will remove it.